components/RefinementList.js

```diff
@@ -108,10 +90,9 @@ RefinementList.propTypes = {
 
 RefinementList.defaultProps = {
   cssClasses: {
-    root: null,
     item: null,
     list: null
   }
 };
 
-module.exports = autoHide(RefinementList);
+module.exports = autoHide(headerFooter(RefinementList));
```

Comment:
I see what you mean but cannot see a solution to do it from the widget/refinement-list.js for example. Do you know how?